.github/workflows/bump.yml

```diff
@@ -41,10 +41,9 @@ jobs:
         uses: "actions/checkout@v4"
       - name: "install rust"
         uses: "dtolnay/rust-toolchain@stable"
-      - uses: "cargo-bins/cargo-binstall@main"
       - name: "install cargo-deny"
         run: |
-          cargo binstall --no-confirm cargo-deny
+          cargo install cargo-deny
```

Comment:
Note: We can ask GitHub runners to cache specific items between runs, if this were helpful. Note sure if that would solve the problem for `cargo binstall`.

Reply:
Fortunately most of the builds are under 30 seconds so maybe not worth fiddling with